content/en/docs/setup/additional-setup/cni/index.md

```diff
@@ -136,6 +136,33 @@ The following table shows the required settings for many common Kubernetes envir
 | Red Hat OpenShift 3.10+ | _(none)_ | _(none)_ |
 | Red Hat OpenShift 4.2+ | `--set components.cni.namespace=kube-system --set values.cni.cniBinDir=/var/lib/cni/bin --set values.cni.cniConfDir=/etc/cni/multus/net.d --set values.cni.chained=false --set values.cni.cniConfFileName="istio-cni.conf" --set values.sidecarInjectorWebhook.injectedAnnotations."k8s\.v1\.cni\.cncf\.io/networks"=istio-cni` | _(none)_ |
 
+#### Istioctl support in 1.4.8+ for OpenShift 4.2+
+
+Istioctl in 1.4.8+ does not support [escaping strings](https://github.com/istio/istio/issues/19196). To install on
+OpenShift 4.2+ you will need to create a file and pass that into istioctl:
+
+cni-annotations.yaml
+```yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  components:
+    cni:
```

Comment:
{{< text yaml >}}
cat <<EOF> cni-annotations.yaml
apiVersion: install.istio.io/v1alpha1
kind: IstioOperator
spec:
  components:
    cni:
      enabled: true
      namespace: kube-system
    values:
      cni:
        chained: false
        cniBinDir: /var/lib/cni/bin
        cniConfDir: /etc/cni/multus/net.d
        cniConfFileName: istio-cni.conf
      sidecarInjectorWebhook:
        injectedAnnotations:
          "k8s.v1.cni.cncf.io/networks": istio-cni
EOF
{{< /text >}}

Reply:
Well, I though I submitted a change suggestion. Trying again. Maybe it doesn't work in this case.

Reply:
Anyway, the thought was to replace that snippet with a text yaml (and get the single line requirement correct) so one can copy and paste to get the file created. This also removed the file name spelling issue.

Then the file is used on the next line.